cmd/noms/migrate/migrate.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package migrate
+
+import (
+	"github.com/attic-labs/noms/go/d"
+	"github.com/attic-labs/noms/go/types"
+	v7types "github.com/attic-labs/noms/go/types"
+)
+
+func Value(source v7types.Value, sourceStore v7types.ValueReadWriter, sinkStore types.ValueReadWriter) (dest types.Value, err error) {
```

Comment:
I wanted the migration code to not be tied to the noms command. I've now moved it to go/migration and renamed the exported function to Migrate.